app-cpp/app355.cpp

```diff
@@ -480,7 +369,388 @@ void SatOrbit::OnPrintTrains(const std::vector<std::vector<OrbitalData>>& trainV
         trainCount++;
     });
 }
+#else
+//----------------------------------------
+#pragma region SatOrbitSingle
+
+class SatOrbitSingle : public SatOrbit
+{
```

Comment:
anonymous namespace vs. "implementation"...
`SatOrbitSingle` is an implementation.
It exists entirely within a `.cpp` and so is inaccessible to anything other than `SatOrbit::Make()`.

But notice it exists within `namespace app355`... and will appear in the `.obj` symboltable as `app355::SatOrbitSingle`, even tho it can never be used by anyone else.

Instead, place it (or any other local implementation) inside anonymous namespace so that it it doesn't appear in the `.obj` symboltable. Ala:
```
namespace {

class SatOrbitSingle : public app355::SatOrbit
{
    // etc...
```